src/leagues/models.py

```diff
@@ -65,8 +65,10 @@ def is_youth(abbreviation: str, name: str) -> bool:
             'Männer', 'Frauen', 'Herren', 'Damen',
             'Hären', 'Dammen', 'Senioren', 'Seniorinnen',
             'Hommes', 'Dames', 'Fraen', ' F-', ' M-',
-            'Inklusion', 'Special Olympics']
-        adult_match_strong = any(n in name for n in adult_name_indicators)
+            'Inklusion', 'Special Olympics',
+            '-F-', '-M-'
+            ]
+        adult_match_strong = any(n in name for n in adult_name_indicators) and "M-Runde" not in name
```

Comment:
this change means: don't strongly match leagues as adult, whose name contains `M-Runde`
it (probably unnecessarily) adds a new condition branch - and thus complexity - complexity to this evaluation statement.
couldn't we instead just solve this by adding `M-Runde` to `youth_name_indicators_direct `?
also this should be backed by a test

Reply:
Should be backed by a test - There was a corner case where `M-Runde` appeared (at least with the old root URL) in the youth league and is also declared as adult league due to the `M` in its `name`.